source/deimos/openssl/ssl.di

```diff
@@ -1944,7 +1944,20 @@ int	SSL_add_dir_cert_subjects_to_stack(STACK_OF!(X509_NAME) *stackCAs,
 //#endif
 }
 
-void	SSL_load_error_strings();
+static if (OPENSSL_VERSION_BEFORE(1, 1, 0, 0))
+{
+    void	SSL_load_error_strings();
+}
+else
+{
+    auto SSL_load_error_strings()()
+    {
+        pragma(inline, true);
+        return OPENSSL_init_ssl(OPENSSL_INIT_LOAD_SSL_STRINGS
+            | OPENSSL_INIT_LOAD_CRYPTO_STRINGS, null);
+    }
```

Comment:
I assume OpenSSL will handle multiple `OPENSSL_init_ssl` correctly ?

Reply:
Yes, there's no other way this would work.